Tribler/Core/leveldbstore.py

```diff
@@ -22,7 +23,7 @@ def get_write_batch_leveldb(self, _):
 
 
 def get_write_batch_plyvel(self, db):
-    from plyveladapter import WriteBatch
+    from .plyveladapter import WriteBatch
```

Comment:
Why did you make these imports relative?

Reply:
I did not, __futurize__ did.  I actually prefer fully qualified imports myself.
* https://www.python.org/dev/peps/pep-0328/

Reply:
Ok, could you make an absolute import out of it then please, to be consistent with the rest of our code base?